recipes/stages/detection/imbalance_ssd.py

```diff
@@ -0,0 +1,35 @@
+_base_ = [
```

Comment:
It seems that there is no need to take the recipe of ssd and yolox separately. 
On the OTE side, there is a way to put data_pipeline in the config, so I recommend you to change it to that way, 
and these seem to be better to erase.